lib/openc_bot/helpers/incremental_search.rb

```diff
@@ -20,6 +20,35 @@ def fetch_data_via_incremental_search(options = {})
         new_highest_numbers
       end
 
+      def fetch_gaps
+        $stderr.puts "#{Time.now.utc.iso8601} - Starting fetch_gaps"
+        current_number = nil 
+        gap_count = 0
+    
+        gap_uid = get_var("highest_gap_uid")
+        gap_uid = sqlite_magic_connection.execute("select min(company_number) from ocdata")[0]["min(company_number)"] if gap_uid.nil?
```

Comment:
can be like this: gap_uid = get_var("highest_gap_uid") || sqlite_magic_connection.execute("select min(company_number) from ocdata")[0]["min(company_number)"]